docs/source/json-api/index.rst

```diff
@@ -32,125 +32,96 @@ We welcome feedback about the JSON API on `our issue tracker
+Often when running the JSON API you'll want to cache queries, especially large queries, to decrease the workload of your underlying ledger.
+The JSON API provides a way to do this by allowing you to specify a database to cache queries (currently PostgreSQL only).
 
-For this reason, you must provide an access token when you start the HTTP JSON API if you're running it against a Ledger API server that requires authentication.
+In this case you can use the ``--query-store-jdbc-config`` flag, an example of which is below. 
 
-Note that this token is used exclusively for maintaining the internal list of known packages and templates, and that it will not be use to authenticate client calls to the HTTP JSON API: the user is expected to provide a valid authentication token with each call.
+.. note:: When you use the Query Store you'll want your first run to specify ``createSchema=true`` so that all the necessary tables are created. After the first run make sure ``createSchema=false`` so that it doesn't attempt to create the tables again.
 
-The HTTP JSON API server requires no access to party-specific data, only access to the ledger identity and package services. A token issued for the HTTP JSON API server should contain enough claims to contact these two services but no more than that. Please refer to your ledger operator's documentation to find out how.
+.. code-block:: shell
 
-Once you have retrieved your access token, you can provide it to the HTTP JSON API by storing it in a file. Give the path to it with the ``--access-token-file`` command line option.
+    daml json-api --ledger-host localhost --ledger-port 6865 --http-port 7575 \
+    --query-store-jdbc-config "driver=org.postgresql.Driver,url=jdbc:postgresql://localhost:5432/test?&ssl=true,user=postgres,password=password,createSchema=false"
 
-If the token cannot be read from the provided path or the Ledger API reports an authentication error (for example due to token expiration), the HTTP JSON API will report the error via logging. The token file can be updated with a valid token, and it will be picked up at the next attempt to send a request.
+.. note:: The JSON API provides many other useful configuration flags, run ``daml json-api --help`` to see all of them.
 
-Example session
-***************
+Authentication and Authorization
+================================
 
-.. code-block:: shell
+The Ledger API and JSON API use the terms Authentication and Authorization to describe different aspects of their operation.
+While these terms may sound similar they mean different things.
+
+The JSON API essentially performs two separate tasks:
 
-    $ daml new iou-quickstart-java --template quickstart-java
-    $ cd iou-quickstart-java/
-    $ daml build
-    $ daml sandbox --wall-clock-time --ledgerid MyLedger ./.daml/dist/quickstart-0.0.1.dar
-    $ daml json-api --ledger-host localhost --ledger-port 6865 --http-port 7575
```

Comment:
I actually think this is the most useful part of the doc :). I was planning to extend this section with `curl` examples showing how to submit a contract and exercise a choice and query the ACS:
```
curl -X POST -H "Authorization: Bearer $JWT" -H "Content-Type: application/json" -d @./create.json http://localhost:7575/v1/create
```

But I am fine removing this section if you think it should not be here.

Reply:
@anthonylusardi-da I like the rest of the changes. Thanks a lot!

Reply:
We still have all of this, it's just parted out differently

https://github.com/digital-asset/daml/pull/7133/files/25338ca7845acd0a7b4b8857497a8a648ba9e39a#diff-acc1415ff18516eee3cce71dc3407d5cR50

Agree adding `curl` examples would be a great addition.

Reply:
While outside of this PR we could put the curl examples after the Authorization with WebSockets section/before HTTP Status Codes. This should generally match the order of operations for people developing DAML applications.

Reply:
sounds good!